_includes/templates/compatibility-page.md

```diff
@@ -1,3 +1,5 @@
+![{{tool.name|escape_once}}]({{tool.logo}}){:.third-party-logo}
```

Comment:
Question: should clicking on the image take you to that wallet/service's website? Or should we include a URL in this page?

Reply:
Bitcoin Core is the fully-worked example page, and for that I had previously linked the name of the wallet in the paragraph that introduces it.  I think that's the best place because it's visually obvious that it's a link.  Let me know if that's satisfactory, or I can also add a link to the image easily.

![2019-04-24-16_10_32_819836136](https://user-images.githubusercontent.com/61096/56690595-d6299180-66ab-11e9-9b05-5158fdcee80e.png)